features/step_definitions/interim_claim_steps.rb

```diff
@@ -18,3 +18,13 @@
 And(/^I enter the effective PCMH date$/) do
   @interim_claim_form_page.interim_fee.effective_pcmh_date.set_date "2016-01-01"
 end
+
+Then(/^I should see interim fee types applicable to a '(.*?)'$/) do |name|
+  expect(@interim_claim_form_page.interim_fee.fee_type_select_names).to include(*expected_interim_fee_for(name))
+end
+
+def expected_interim_fee_for(case_type_name)
```

Comment:
**[nit pick/not a blocker]** Should the method name be `expected_interim_fee<s>_for` if the method is return a _list_ of **fees**?